src/docs/Components/Modals/code.mdx

```diff
@@ -102,6 +102,12 @@ In cases where it is not necessary, you can use a modal without heading, as foll
 
 <Preview path="withouttitle" />
 
+### Scrolling long content
+
+In some cases, the content paragraph can be bigger than the modal's viewport. A dropped shadow on the top of the footer is used to indicate to the user that this is a scrollable content.
+
+<Preview path="big-content" />
```

Comment:
Do we have the right shadow property here? Seems harder than the design file

Reply:
probably not! 
Would you please send me the link to make it consistent

Reply:
https://share.goabstract.com/78533af2-8f38-4e31-b3d1-b6dbf88effb3?collectionLayerId=a25c68f5-bf2d-4883-b237-1bbf24e202af&mode=build

Reply:
thanks @nherwegh,  ready for you to review